packages/common-common/src/rabbitmq/scripts/publishRmqMessage.ts

```diff
@@ -0,0 +1,34 @@
+import { publishRmqMsg } from "common-common/src/rabbitmq/util";
+import {EventKind, ITransfer, IEventData} from "chain-events/src/chains/aave/types";
+import {CWEvent, SupportedNetwork} from "chain-events/src";
+import {v4 as uuidv4} from 'uuid';
+import {RascalExchanges, RascalRoutingKeys} from "common-common/src/rabbitmq";
+import {RABBITMQ_API_URI} from "commonwealth/server/config";
+
+async function publishRmqMessage() {
+  const ceData: ITransfer = {
+    kind: EventKind.Transfer,
+    tokenAddress: uuidv4(),
+    from: uuidv4(),
+    to: uuidv4(),
+    amount: uuidv4()
+  }
+  // // create a fake aave-transfer event
+  const chainEvent: CWEvent<IEventData> = {
+    blockNumber: Math.floor(Math.random() * 1000000),
+    data: ceData,
+    network: SupportedNetwork.Aave,
+    chain: 'aave'
+  }
+
+  const publishJson = await publishRmqMsg(
+    RABBITMQ_API_URI,
+    RascalExchanges.ChainEvents,
+    RascalRoutingKeys.ChainEvents,
+    chainEvent
+  );
+
```

Comment:
either we should wrap this whole async function in a `try catch` and log the error. Or (maybe better) `publishRmqMsg` should have its logic wrapped in a try catch, throwing an error if something went wrong.

Reply:
`publishRmqMsg` is not meant to be used in production and mostly a convenience method for testing/other scripts so in this case I do want it to throw and since this file is just a script to be used with yarn it will simply log the error and exit which is also the intended behaviour.